src/Datadog.Trace/TraceConstants.cs

```diff
@@ -0,0 +1,7 @@
+namespace Datadog.Trace
+{
+    internal static class TraceConstants
+    {
+        public const ulong MaxTraceId = 9_223_372_036_854_775_807; // 2^63-1
```

Comment:
In a recent PR, Bob added a `TracerConstants` class. Do you think this makes sense to put in that class instead so that we can consolidate?